homeassistant/components/aussie_broadband/config_flow.py

```diff
@@ -93,17 +83,34 @@ async def async_step_service(
                 for s in self.services
                 if s["service_id"] == user_input[CONF_SERVICE_ID]
             )
-            self.data[CONF_SERVICE_ID] = service["service_id"]
-
-            existing_entry = await self.async_set_unique_id(self.data[CONF_SERVICE_ID])
-            if existing_entry is not None:
-                return self.async_abort(reason="already_configured")
-
-            return self.async_create_entry(title=service["description"], data=self.data)
+            return await self.create_entry(service)
 
         service_options = {s["service_id"]: s["description"] for s in self.services}
-        schema = vol.Schema({vol.Required(CONF_SERVICE_ID): vol.In(service_options)})
         return self.async_show_form(
             step_id="service",
-            data_schema=schema,
+            data_schema=vol.Schema(
+                {vol.Required(CONF_SERVICE_ID): vol.In(service_options)}
+            ),
         )
+
+    async def async_step_reauth(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle reauth."""
+        self._reauth = True
+        return await self.async_step_user(user_input)
+
+    async def create_entry(self, service):
```

Comment:
Should we prefix with `_` to indicate a private member?

```suggestion
    async def _create_entry(self, service):
```